src/stores/media-library-item-store.js

```diff
@@ -0,0 +1,35 @@
+import Database from './database.js';
+import { validate } from '../domain/validation.js';
+import { createTagsPipelineQuery } from '../utils/tag-utils.js';
+import { mediaLibraryItemDbSchema } from '../domain/schemas/media-library-item-schemas.js';
+
+class MediaLibraryItemStore {
+  static get inject() { return [Database]; }
+
+  constructor(db) {
+    this.collection = db.mediaLibraryItems;
+  }
+
+  getMediaLibraryItemsByConditions(conditions, { session } = {}) {
+    return this.collection.find({ $and: conditions }, { session }).toArray();
+  }
+
+  getMediaLibraryItemTagsMatchingText(text) {
```

Comment:
getMediaLibraryItem**s**TagsMatchingText

Reply:
It's debatable. I named this after `getDocumentTagsMatchingText` that does the same for document tags (actually not: documents tags). The function does not return a list of MediaLibraryItems but a list of MediaLibraryItemTags (tags that are set on a MediaLibraryItems). I'd personally prefer to keep it like that, but I'm also not very passionate about it.